www/index.js

```diff
@@ -665,14 +665,17 @@ function setModel(overview, target, id) {
 function changeModel(version, overview, title, base_url) {
   const entry = overview.profiles[title];
   if (entry) {
-    fetch(`${base_url}/${entry.target}/${entry.id}.json`, {
+    fetch(`${base_url}/targets/${entry.target}/profiles.json`, {
```

Comment:
It seems that instead of the individual JSON files, the OpenWrt generated profiles.json files are used. Those file contain all models of the target. This OK.

But we would still need to create the overview.json file. No?

Reply:
It's now outomatocally created

Reply:
I am not sure what  automatic means. Is the overview.json now generated by the OpenWrt build process?

Reply:
Correct

Reply:
Is that code now part of https://github.com/openwrt/openwrt ?

Reply:
No it's part of the way we deploy downloads.openwrt.org